libraries/radwordsprocessing/formats-and-conversion/rtf/rtfformatprovider.md

```diff
@@ -47,6 +50,9 @@ And here is how you can import a document from string containing the RTF documen
 #### __[C#] Example 2: Import document from a string__
 
 {{region cs-radwordsprocessing-formats-and-conversion-rtf-rtfformatprovider_1}}
+	using Telerik.Windows.Documents.Flow.FormatProviders.Rtf;
+	using Telerik.Windows.Documents.Flow.Model;
```

Comment:
Do not add usings to the snippet - it won't build when directly pasted. Instead, use full class names.